.github/workflows/ci_unit.yml

```diff
@@ -87,6 +138,6 @@ jobs:
       - run: |
           julia --project=docs -e '
             using Documenter: DocMeta, doctest
-            using RegistryCI
-            DocMeta.setdocmeta!(RegistryCI, :DocTestSetup, :(using RegistryCI); recursive=true)
```

Comment:
Is there a reason we no longer need this?

Reply:
it wasn't used on main either; I removed it since I got confused how to update it, then realized it wasn't needed